app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -114,6 +114,9 @@ $include-html-global-classes: $include-html-classes;
 // $oil         : #333333;
 // $jet         : #222222;
 // $black       : #000000;
+ $share-purple: #AD1BA1;
```

Comment:
Color `#AD1BA1` should be written as `#ad1ba1`<br>Line should be indented 0 spaces, but was indented 1 spaces